resources/appimage/AppDir/bin/cryptomator.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+cd $(dirname $0)
+
+BUILD_NUMBER=$(cat ./build.number)
+
+# determine GTK version
+GTK2_CMD_OUT="echo" #just a dummy
+GTK3_CMD_OUT="echo" #just a dummy
+if [ -n $(which dpkg) ]
+    then #do stuff for debian based things
```

Comment:
have `if` and `then` in a single line (feels more familiar, if one is used to other languages)

```suggestion
if [ -n $(which dpkg) ]; then #do stuff for debian based things
```

Reply:
Resolved in 23a7df83873519602bfe6a0ec300323bebc4f78b.